docs/APIs/tasks/events.md

```diff
@@ -1,9 +1,9 @@
 # About the mbed OS event loop
 
-One of the optional mbed OS features is an event loop mechanism that can be used to defer the execution of code to a different context. In particular, a common use of an event loop is to postpone the execution of a code sequence from an interrupt handler to a user context. This is useful because of the specific constraints of code that runs in an interrupt handler:
+One of the optional mbed OS features is an event loop mechanism that you can use to defer the execution of code to a different context. In particular, a common use of an event loop is to postpone the execution of a code sequence from an interrupt handler to a user context. This is useful because of the specific constraints of code that runs in an interrupt handler:
 
 - The execution of certain functions (notably some functions in the C library) is not safe.
-- Various RTOS objects and functions can't be used from an interrupt context.
+- You cannot use various RTOS objects and functions from an interrupt context.
```

Comment:
Not sure I agree with this one. I think the emphasis of the sentence is the "various", meaning several different kinds of objects. A non-native speaker may well understand "various" in the active version to mean several at once, not just different ones. Not sure the passive solves the issue, but pretty sure the current active doesn't, either. Perhaps there's a third alternative.